py/plugins/compat-oai/tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from genkit.core.typing import (
+    GenerateRequest,
+    Message,
+    Role,
+    TextPart,
+)
+from genkit.plugins.compat_oai.models.model_info import GPT_4
+from genkit.plugins.compat_oai.typing import OpenAIConfig
+
+
+@pytest.fixture
```

Comment:
for educational purposes, why make it a fixture instead of directly calling as a function?

Reply:
I figured out how they work, but trying to understand what benefits they offer over just calling the function:

```py
def test_something():
    sample = sample_request()
    ...
```

One downside to fixtures that I'm experiencing right now, is that they make it very hard to navigate the code. I mean, for a function I can ctrl+click on the function in vscode and it will take me to where it's declared. With fixtures it doesn't work. I have to search by fixture name... and if the fixture is declared in a separate file it's even harder to find... and then if fixtures depend on other fixtures... 

So, I want to make sure that the benefits of using fixtures outweigh the downsides.